src/StronglyTypedIds/Templates/ObjectId/ObjectId_SystemTextJsonConverter.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+        class TESTIDSystemTextJsonConverter : System.Text.Json.Serialization.JsonConverter<TESTID>
+        {
+            public override TESTID Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options)
+            {
+                var result = reader.GetString();
+                return string.IsNullOrEmpty(result) ? TESTID.Empty : new TESTID(new MongoDB.Bson.ObjectId(result));
```

Comment:
I think there's a discrepancy in the way you're handling null/empty string in these serializers. Some return `null`, some return `TESTID.Empty`, and some defer to the default handler

Reply:
Ups, you have right. I will correct it.

Reply:
Hi @marspion, so it looks like you've made add the serializers return `TESTID.Empty` if the object can't be parsed? I'm not sure if that's the best behaviour. I'd expect a serializer to throw if it can't be parsed I think? Otherwise you could get unexpected behaviour?